databases/week1/w1.sql

```diff
@@ -0,0 +1,17 @@
+SELECT COUNT(task.id) FROM task;
```

Comment:
Please add the comments of each task, it's much easier to follow the intended purpose of each query :)